src/collective/solr/setuphandlers.py

```diff
@@ -63,3 +65,19 @@ def migrateTo4(context):
             logger.info("Added new behavior to {}".format(type_id))
 
     logger.info("Migrated to version 4")
+
+
+def migrate_to_5(context):
```

Comment:
@mpeeters isn't it sufficient to just add import_step="plone.app.registry" like here?

https://github.com/collective/collective.solr/pull/302/files#diff-3575f29c0bad1431372ca7d0dceaad5cf8f0b7c5806a54a1b809a44e00375fe3R41

Reply:
@tisto if I remember correctly by reload registry some parameters defined directly in plone controlpanel may be overrided by default values. I had this bad experience in the past on numerous projects and this is the reason why I always do upgrade steps that only interract with the registry records that I want to add, modify or remove.

Reply:
I second @mpeeters approach.
We also had some issues recently https://github.com/plone/plone.app.registry/issues/58

Reply:
@ale-rt @mpeeters ok, will merge this PR first and then change my approach to yours. I recall that we had issues with registry entries as well. I just wanted to double-check.

It would be great if you folks would find time to have a quick look at my PR:

https://github.com/collective/collective.solr/pull/302

I'd like to merge those two and then do another alpha release.